modules/ztp-common-expected-errors.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-factory-install-clusters.adoc
+:_content-type: CONCEPT
+[id="common-expected-errors_{context}"]
+= Common and expected errors
+
+A common issue that may occur during the ZTP pipelines run is a failure of the download the code and check hub stage.
```

Comment:
no such stage "download the code and check hub"
Is this 2 stages?

Reply:
no - I updated diagram per feedback to remove that download the code bit forgot to remove here